dashboard/test/ui/features/teacher_tools/pl_sections.feature

```diff
@@ -1,45 +1,16 @@
 @no_mobile
 Feature: Professional learning Sections
 
-  Scenario: Create new professional learning section as levelbuilder
```

Comment:
Just so I'm clear, it looks like the test for "as a levelbuilder" was removed but creating PL sections is still covered by other tests in this file?